cli/src/main.rs

```diff
@@ -37,26 +43,76 @@ mod machine;
 mod storage;
 mod subnet;
 
+const DEFAULT_NETWORK_CONFIG_PATH: &str = "~/.config/recall/networks.toml";
```

Comment:
i dont think we should `~` + shell expand to get the home dir, and i think `.recall` should be the dir name, not a generic `.config`.

maybe something like:
```rust
use std::path::{Path, PathBuf};

fn default_network_config_path() -> PathBuf {
    dirs::home_dir()
        .expect("Home directory not found")
        .join(".recall/networks.toml") # or .recall/config.toml"
}
```

Reply:
`~/.config` is a standard practice though. i have about 50 dirs under ~/.config on my machine. it's nicer than polluting home dir with a bunch of dot dirs

Reply:
I thought we want to allow users to have `~` in the paths. If so then it would be consistent to have a constant that we show in the help message and use in the code.

I don't mind to switch to `~/.recall` instead of `~/.config/recall`. Just out of curiosity why? 🙂 My observation that the trend is going to `~/.config/xyz`, to keep the home dir "clean".

Reply:
Agreed `~/.config/recall` would not clutter the dev setup. but having the `~` in that constant will make it incompatible with Windows, right? I'm not sure if the SDK currently works with Windows but if we can avoid the explicit tilde in the path then it would be great

Reply:
@avichalp The `~` expansion is handled by `shellexpand` crate that apparently handles Windows paths too. So it should work. Another option would be `$HOME/.config/recall/networks.toml`. `shellexpand` can handle this too. I'd just go with the simple `~` and improve later if needed. No preferences from my side, though.